Keyboards/LanguageKeyboards/Hebrew/HEInterfaceVariables.swift

```diff
@@ -1,51 +1,32 @@
-/**
- * Constants and functions to load the Hebrew Scribe keyboard.
- *
- * Copyright (C) 2024 Scribe
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
- */
+//  HEInterfaceVariablesTemp.swift
+//  Scribe
 
 import UIKit
 
 public enum HebrewKeyboardConstants {
-  static let defaultCurrencyKey = "$"
-  static let currencyKeys = ["$", "€", "£", "¥"]
+  static let defaultCurrencyKey = "₪"
+  static let currencyKeys = ["₪", "€", "£", "$"]
 
-  // Alternate key vars.
-  static let keysWithAlternates = [""]
-  static let keysWithAlternatesLeft = [""]
-  static let keysWithAlternatesRight = [""]
+  
 }
 
 struct HebrewKeyboardProvider: KeyboardProviderProtocol {
   // iPhone keyboard layouts.
   static func genPhoneLetterKeys() -> [[String]] {
     return KeyboardBuilder()
-      .addRow(["פ", "ם", "ן", "ו", "ט", "א", "ר", "ק", "delete"])
+      .addRow(["פ", "ם", "ן", "ו", "ט", "א", "ר", "ק", "❌"])
```

Comment:
What's the goal of `X` here instead of `delete`? I think we'll be able to figure out a right delete icon for this and swap it in, if that's what you're worried about :)

Reply:
Yea it was as a delete button, I'll replace with "delete" for the time being